src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -171,8 +217,18 @@ public void periodic() {
                 } else {
                         updateRotationTarget = true;
                 }
+                
+                positionMeters += frontLeftModule.getDriveVelocity() * Timer.getFPGATimestamp() - lastUpdateTime; // (m / s) * delta t = m
```

Comment:
I think we need to track the position of each module based on some of the other examples I saw